internal/transform/operation.go

```diff
@@ -0,0 +1,345 @@
+		if err != nil {
+			return Details{}, err
+		}
+
+		outputDetails.Price = parsedPrice
+		outputDetails.PriceR = Price{
+			Numerator:   int32(op.Price.N),
+			Denominator: int32(op.Price.D),
+		}
+		addAssetDetailsToOperationDetails(&outputDetails, op.Buying, "buying")
+		addAssetDetailsToOperationDetails(&outputDetails, op.Selling, "selling")
+
+	case xdr.OperationTypeCreatePassiveSellOffer:
+		op := operation.Body.MustCreatePassiveSellOfferOp()
+		outputDetails.Amount = utils.ConvertStroopValueToReal(op.Amount)
+		parsedPrice, err := strconv.ParseFloat(op.Price.String(), 64)
+		if err != nil {
+			return Details{}, err
+		}
+
+		outputDetails.Price = parsedPrice
+		outputDetails.PriceR = Price{
+			Numerator:   int32(op.Price.N),
+			Denominator: int32(op.Price.D),
+		}
+		addAssetDetailsToOperationDetails(&outputDetails, op.Buying, "buying")
+		addAssetDetailsToOperationDetails(&outputDetails, op.Selling, "selling")
+
+	case xdr.OperationTypeSetOptions:
+		op := operation.Body.MustSetOptionsOp()
+
+		if op.InflationDest != nil {
+			outputDetails.InflationDest = op.InflationDest.Address()
+		}
+
+		if op.SetFlags != nil && *op.SetFlags > 0 {
+			addOperationFlagToOperationDetails(&outputDetails, int32(*op.SetFlags), "set")
+		}
+
+		if op.ClearFlags != nil && *op.ClearFlags > 0 {
+			addOperationFlagToOperationDetails(&outputDetails, int32(*op.ClearFlags), "clear")
+		}
+
+		if op.MasterWeight != nil {
+			outputDetails.MasterKeyWeight = int32(*op.MasterWeight)
+		}
+
+		if op.LowThreshold != nil {
+			outputDetails.LowThreshold = int32(*op.LowThreshold)
+		}
+
+		if op.MedThreshold != nil {
+			outputDetails.MedThreshold = int32(*op.MedThreshold)
+		}
+
+		if op.HighThreshold != nil {
+			outputDetails.HighThreshold = int32(*op.HighThreshold)
+		}
+
+		if op.HomeDomain != nil {
+			outputDetails.HomeDomain = string(*op.HomeDomain)
+		}
+
+		if op.Signer != nil {
+			outputDetails.SignerKey = op.Signer.Key.Address()
+			outputDetails.SignerWeight = int32(op.Signer.Weight)
+		}
+
+	case xdr.OperationTypeChangeTrust:
+		op := operation.Body.MustChangeTrustOp()
+		addAssetDetailsToOperationDetails(&outputDetails, op.Line, "")
+		outputDetails.Trustor = sourceAccountAddress
+		outputDetails.Trustee = outputDetails.AssetIssuer
+		outputDetails.Limit = utils.ConvertStroopValueToReal(op.Limit)
+
+	case xdr.OperationTypeAllowTrust:
+		op := operation.Body.MustAllowTrustOp()
+		addAssetDetailsToOperationDetails(&outputDetails, op.Asset.ToAsset(sourceAccount.ToAccountId()), "")
+		outputDetails.Trustee = sourceAccountAddress
+		outputDetails.Trustor = op.Trustor.Address()
+		shouldAuth := xdr.TrustLineFlags(op.Authorize).IsAuthorized()
+		outputDetails.Authorize = shouldAuth
+
+	case xdr.OperationTypeAccountMerge:
+		aid := operation.Body.MustDestination().ToAccountId()
+		outputDetails.Account = sourceAccountAddress
+		outputDetails.Into = aid.Address()
+
+	case xdr.OperationTypeInflation:
+		// inflation is deprecated
```

Comment:
This needs to be handled, since you'll be ingesting operations from genesis, and you don't want to just drop all the inflation ops.